wear/src/main/java/com/android/gpstest/wear/MainActivity.kt

```diff
@@ -162,7 +167,7 @@ class MainActivity : ComponentActivity() {
 
 @OptIn(ExperimentalCoroutinesApi::class)
 @Composable
-fun WearApp(signalInfoViewModel: SignalInfoViewModel) {
+fun WearApp(context: Context, signalInfoViewModel: SignalInfoViewModel) {
```

Comment:
`Application.app` is a context, so you don't need to pass in a `Context` here - just use `Application.app` below where you are currently using `context`.

Reply:
> `Application.app` is a context, so you don't need to pass in a `Context` here - just use `Application.app` below where you are currently using `context`.

Ok, thanks!